src/components/primitives/Slider/SliderThumb.tsx

```diff
@@ -19,17 +19,20 @@ const SliderThumb = ({ children, ...props }: ISliderProps) => {
     SliderThumb: {
       position: 'absolute',
       display: 'flex',
-      left: sliderOffset,
+      // @ts-ignore - cheking for undefined in the context itself.
```

Comment:
It does not work like that. You will have to add checks here too. Please avoid @ts-ignore unless it is unavoidable.